app/src/main/res/values/styles.xml

```diff
@@ -74,7 +74,9 @@
         <item name="android:insetBottom">0dp</item>
         <item name="android:letterSpacing">0</item>
         <item name="android:minHeight">48dp</item>
-        <item name="android:paddingHorizontal">@dimen/default_padding</item>
+        <!-- <item name="android:paddingHorizontal">@dimen/default_padding</item> -->
```

Comment:
@rawmain why do we need to comment these fields?

Reply:
Remove old unnecessary comments. Thanks.